diff --git a/setup.py b/setup.py
index c2c3e12..e54c4ff 100644
--- a/setup.py
+++ b/setup.py
@@ -282,10 +282,8 @@ setuptools.setup(
     ],
     python_requires='>=3.6',
     install_requires=[
-        'numpy',
-        'packaging',
-        'pydot==1.4.1',
-        'h5py>=2.8.0'],
+        'packaging'
+    ],
     extra_require={'full': ['tensorflow>=1.12.0,<2']},
     # cmdclass={'build_ext': BuildExt},
     cmdclass={
